examples/register_and_change_username.php

```diff
@@ -32,25 +34,30 @@
 $commandMap = [
     RegisterUser::class => [
         'handler' => function (array $state, Message $message) use (&$factories): AggregateResult {
-            return \Prooph\MicroExample\Model\User\registerUser($state, $message, $factories['emailGuard']());
+            return User\registerUser($state, $message, $factories['emailGuard']());
         },
         'definition' => UserAggregateDefinition::class,
     ],
+    'handler' => function (array $state, Message $message): AggregateResult {
```

Comment:
I modified this PR a little bit, if there is a problem, let's create another PR.